docs/cookbook/recipe_custom_action.rst

```diff
@@ -46,30 +51,30 @@ Either by using XML:
 
 .. code-block:: xml
 
-        <!-- src/AppBundle/Resources/config/admin.xml -->
+        <!-- src/Resources/config/admin.xml -->
 
-        <service id="app.admin.car" class="AppBundle\Admin\CarAdmin">
+        <service id="app.admin.car" class="App\Admin\CarAdmin">
             <tag name="sonata.admin" manager_type="orm" group="Demo" label="Car" />
             <argument />
-            <argument>AppBundle\Entity\Car</argument>
-            <argument>AppBundle:CRUD</argument>
+            <argument>App\Entity\Car</argument>
+            <argument>App:CRUD</argument>
```

Comment:
This does not work:

```yaml
live2:
    path: /live2/{id}
    controller: App:Live:index
```

![screenshot_3](https://user-images.githubusercontent.com/2865341/37240186-c8690948-2450-11e8-8193-0df0f08aceff.png)

Additionally it's also going to be deprecated in 4.1: https://symfony.com/blog/new-in-symfony-4-1-deprecated

Reply:
That is service argument and not route configuration, but I got your point, just wrong example :smile:

Reply:
Well it becomes route config later :) But yeah, had some brain lag - would've been better to showcase on an actual admin class